src/olympia/reviews/tests/test_models.py

```diff
@@ -109,6 +109,53 @@ def test_soft_delete_replies_are_hidden(self):
         # Unfiltered should have them all still.
         assert Review.unfiltered.count() == 3
 
+    def test_moderator_delete(self):
+        moderator = user_factory()
+        review = Review.objects.get(pk=1)
+        review.update(editorreview=True)
+        review.reviewflag_set.create()
+        review.moderator_delete(user=moderator)
+
+        review.reload()
+        assert ActivityLog.objects.count() == 1
+        assert not review.reviewflag_set.exists()
+
```

Comment:
Yeah. I've added a test that the logging in `delete()` is using the right user, since that's the only use of this parameter for now.